### DISCUSSION TITLE
A question about undersampling mask in singlecoil-knee dataset

### DISCUSSION THREAD

**Discussion by weijinbao1998** [facebookresearch/fastMRI#250] [Q&A] [answered]
The original U-net network you posted was applied 4x Random Mask to train and evaluate.But I change the mask size to only random sample a certain location but still in 4x Random Mask. Then I made some adjustment to the undersampled data in file( UnetDataTransform), and ultimately improve the metrics.
Does the improvement I made countable and convincible?Or we can only use the original 4x Random Mask to do some improvements.
Look forward to your reply!

**Comment by mmuckley** [comment, marked as answer]
Hello @1025754824, there have been many improvements over the original method, as you can see on the fastMRI leaderboard https://fastmri.org/leaderboards/.

In terms of convincible - I'm a little less able to help there. You might be able to open some general discussion here or email some researchers in the field.
